project3/index.html

```diff
@@ -0,0 +1,232 @@
+                        style="stroke-width: 2px; fill: rgb(250, 250, 250);" />
+                </svg> </div>
+            <h1>Super hero registry</h1>
+            <p class="quote">With great power there must also come-great resposibility</p>
+        </header>
+    </section>
+    <div class="form">
+
+        <fieldset class="identity">
+            <h2>Identity </h2>
+            <label for="real-name">Real Name</label>
+            <span> <input id="real-name" type="text">
+            </span>
+
+            <label for="alias">Alias</label>
+            <span> <input id="alias" type="text">
+            </span>
+
+            <label for="email">Email</label>
+            <span> <input id="email" type="email">
+            </span>
+        </fieldset>
+
+
+        <fieldset class="personalcharacteristics-left-column">
+            <h2> Personal Characteristics</h2>
+            <label for="age">Age</label>
+            <span><input id="age" type="number"></span>
+
+            <span>
+                <legend>Category</legend>
+                <p class="categoryinfo">Please select as many as apply to your origin and how you gained your powers
+                </p>
+            </span>
+
+            <div class="options">
+                <input id="alien" type="checkbox">
+                <label for="alien">Alien</label>
+            </div>
+            <span>
+                <input id="cyborg" type="checkbox">
+                <label for="cyborg">Cyborg</label>
+            </span>
+            <span>
+                <input id="experiment" type="checkbox">
+                <label for="experiment">Experiment</label>
+            </span>
+            <span>
+                <input id="human" type="checkbox">
+                <label for="human">Human</label>
+            </span>
+            <span>
+                <input id="inhuman" type="checkbox">
+                <label for="inhuman">Inhuman</label>
+            </span>
+            <span>
+                <input id="mutant" type="checkbox">
+                <label for="mutant">Mutant</label>
+            </span>
+        </fieldset>
+
+        <fieldset class="personalcharacteristics-right-column">
+
+            <legend>Gender</legend>
+
+            <span>
+                <input name="gender" id="female" type="radio">
```

Comment:
Per [requirements](https://mica-web.github.io/gd-220/proj3.html#requirements), this field should be **required**; note that for sets of inputs these radio inputs, you can _either_ make each one required or just one and it will apply to all:
```
<input name="gender" id="female" type="radio" required>
```